library/core/src/slice/sort/unstable/mod.rs

```diff
@@ -1,10 +1,13 @@
 //! This module contains the entry points for `slice::sort_unstable`.
 
 use crate::mem::SizedTypeProperties;
+use crate::ops::{Range, RangeBounds};
+use crate::slice::sort::select::partition_at_index;
 #[cfg(not(any(feature = "optimize_for_size", target_pointer_width = "16")))]
 use crate::slice::sort::shared::find_existing_run;
 #[cfg(not(any(feature = "optimize_for_size", target_pointer_width = "16")))]
 use crate::slice::sort::shared::smallsort::insertion_sort_shift_left;
+use crate::slice::{self};
```

Comment:
Nit, unusual import style

```suggestion
use crate::slice;
```